simgui.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
Same as the other file, maybe we could add to the `.gitignore`, unless we actually want these.

Reply:
Done.